pkg/network/placement_configuration.go

```diff
@@ -0,0 +1,76 @@
+package network
+
+import (
+	"reflect"
+
+	cnao "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func fillDefaultsPlacementConfiguration(conf, previous *cnao.NetworkAddonsConfigSpec) []error {
+	if conf.PlacementConfiguration == nil || conf.PlacementConfiguration.Infra == nil || conf.PlacementConfiguration.Workloads == nil {
+		if previous != nil && previous.PlacementConfiguration != nil {
+			conf.PlacementConfiguration = previous.PlacementConfiguration
+			return []error{}
+		}
+
+		conf.PlacementConfiguration = &cnao.PlacementConfiguration{
+			Infra: &cnao.Placement{
+				Affinity: corev1.Affinity{
+					PodAntiAffinity: &corev1.PodAntiAffinity{
+						PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
+							corev1.WeightedPodAffinityTerm{
+								PodAffinityTerm: corev1.PodAffinityTerm{
+									LabelSelector: &metav1.LabelSelector{
+										MatchExpressions: []metav1.LabelSelectorRequirement{
+											metav1.LabelSelectorRequirement{
+												Key:      "control-plane",
+												Operator: metav1.LabelSelectorOpIn,
+												Values:   []string{"mac-controller-manager"},
```

Comment:
Now with a single replica in KMP, we can drop this affinity completely. It would make this code far more readable too

Reply:
Right, removed.